### PR TITLE
Adding Release notes shadow details

### PR DESCRIPTION
**Pull request by pmmalinov01**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:
Adding Release Notes Shadows to the release team page.

#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:

/assign @savitharaghunathan @guineveresaenger @annajung  @erismaster @divya-mohan0209 @kikisdeliveryservice
/cc @cici37 @Damans227 @sladyn98 @simcard0000 

/priority important-soon

Welcome and smooth sailing 🚢 🎉

### CODE REVIEW

**Comment by pmmalinov01** [comment]
Why PROW did not assign? What have I missed? Lets try again.
/assign @savitharaghunathan @guineveresaenger @annajung  @erismaster @divya-mohan0209  @kikisdeliveryservice

File: releases/release-1.22/release-team.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 | CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | |
 | Bug Triage | Menna Elmasry ([@monzelmasry](https://github.com/MonzElmasry) / Slack: `@Menna`) | |
 | Docs | Victor Palade ([@pi-victor](https://github.com/pi-victor)) / Slack: `@Victor PI`) | |
-| Release Notes | Pavel Malinov ([@pmmalinov01](https://github.com/pmmalinov01) / Slack: `@Pavel Malinov`) | |
+| Release Notes | Pavel Malinov ([@pmmalinov01](https://github.com/pmmalinov01) / Slack: `@Pavel Malinov`) | Cici Huang ([@cici37](https://github.com/cici37) / Slack: `cici37`), Daman Arora ([@Damans227](https://github.com/Damans227) / Slack: `@Daman Arora`), Sladyn Nunes ([@sladyn98](https://github.com/sladyn98) / Slack: `@Sladyn Nunes`), Simran Thind ([@simcard0000](https://github.com/simcard0000) / Slack: `@Sims`) |
```
Comment (annajung) [inline releases/release-1.22/release-team.md:10]: missing `@` in front of `cici37`

```suggestion
| Release Notes | Pavel Malinov ([@pmmalinov01](https://github.com/pmmalinov01) / Slack: `@Pavel Malinov`) | Cici Huang ([@cici37](https://github.com/cici37) / Slack: `@cici37`), Daman Arora ([@Damans227](https://github.com/Damans227) / Slack: `@Daman Arora`), Sladyn Nunes ([@sladyn98](https://github.com/sladyn98) / Slack: `@Sladyn Nunes`), Simran Thind ([@simcard0000](https://github.com/simcard0000) / Slack: `@Sims`) |
```

Comment (pmmalinov01) [reply releases/release-1.22/release-team.md:10]: Thanks for catching that.

**Comment by Damans227** [comment]
Looks like prow missed out on me to be added as a reviewer as I am not an org member yet. @pmmalinov01 Does this matter ?

**Comment by pmmalinov01** [comment]
@Damans227 Yep, that is the next important step we will take, adding you and @simcard0000 as Kubernetes org members.

**Comment by cici37** [comment]
/lgtm
Thank you for having me :)

**Comment by annajung** [comment]
Thanks Pavel!

Welcome to the release team everyone! looking forward to working with you all.
/lgtm

**Pull request merged**